frontend/src/store/modules/users/state.js

```diff
@@ -7,5 +7,6 @@ export default {
         facebook_url: '',
         twitter_url: '',
         instagram_url: '',
-    }
+    },
+    places: []
```

Comment:
state should have reviews: {} and profile only, lists will be fetched from userList module